src/server/pingserver-rs/src/session.rs

```diff
@@ -126,14 +110,12 @@ impl Session {
         match read_result {
             Ok(Some(0)) => Ok(Some(0)),
             Ok(Some(bytes)) => {
-                let _ = self
-                    .metrics
-                    .increment_counter(&Stat::TcpRecvByte, bytes.try_into().unwrap());
+                increment_counter_by!(&Stat::TcpRecvByte, bytes as u64);
```

Comment:
Do we need to track tls read/write aside from tcp read/write? Or only one or the other depending on the socket type?

Reply:
I think we should track both if we have TLS enabled. Some of this will change in an upcoming PR. I will be adding a thin wrapper around TcpStream that ensures all read/write stats are tracked there - will reduce need to increment independently in admin/worker

Reply:
Created #320 to track this